docs/framework/data/adonet/ef/language-reference/method-based-query-syntax-examples-navigating-relationships.md

```diff
@@ -47,5 +47,5 @@ Navigation properties in the [!INCLUDE[adonet_ef](../../../../../../includes/ado
  [!code-vb[DP L2E Examples#WhereNavProperty](../../../../../../samples/snippets/visualbasic/VS_Snippets_Data/DP L2E Examples/VB/Module1.vb#wherenavproperty)]  
   
 ## See also
-- [Navigation Properties](https://msdn.microsoft.com/library/41e1e6b9-8a57-467d-99d9-1857d2ca2ea5)
+- [Navigation Properties](https://docs.microsoft.com/previous-versions/dotnet/netframework-4.0/bb738520(v=vs.100))
```

Comment:
There's a second topic that uses the same link. Make sure we change that one as well to use this same link below.

```suggestion
- [Relationships, navigation properties and foreign keys](/ef/ef6/fundamentals/relationships)
```